test/expect/TestJit.test_concat_fusion_invariant_cuda.expect

```diff
@@ -0,0 +1,17 @@
+graph(%0 : Float(2, 2)
+      %1 : Float(2, 2)
+      %2 : Float(4, 2)) {
+  %3 : int = prim::Constant[value=1]()
+  %4 : Float(2, 2) = aten::sub(%0, %1, %3)
```

Comment:
Actually why isn't this node fused? I can't see any reason why it shouldn't.

Reply:
we special case aten::add(Tensor, Tensor, Scalar) in the graph fuser to fuse it. We should do the same thing for sub but we don't do that right now: https://github.com/pytorch/pytorch/blob/f6eb966fd2ae22d1a96678f595b11cffa06e7fd0/torch/csrc/jit/passes/graph_fuser.cpp#L162-L164